### DISCUSSION TITLE
Connect to Sybase SQL Anywhere from Windows

### DISCUSSION THREAD

**Discussion by victor-ivanov** [FreeTDS/freetds#522] [Q&A] [answered]
I got binaries for Windows from https://ci.appveyor.com/project/FreeTDS/freetds/history 
When running `tsql -C` I get 
```
Compile-time settings (established with the "configure" script)
                            Version: freetds v1.5.dev.20231217
             freetds.conf directory: c:
     MS db-lib source compatibility: yes
        Sybase binary compatibility: no
                      Thread safety: yes
                      iconv library: yes
                        TDS version: auto
                              iODBC: no
                           unixodbc: no
              SSPI "trusted" logins: yes
                           Kerberos: no
                            OpenSSL: yes
                             GnuTLS: no
                               MARS: yes
```

Regarding this specific line 
```
Sybase binary compatibility: no
```
Does it mean that this binaries was compiled in the mode excluding Sybase? Or is it something I have to configure on my machine?

Currently it hangs silently when I try to connect to SQL Anywhere DB

**Comment by freddy77** [comment, marked as answer]
Don't worry, the
```
Sybase binary compatibility: no
```
is mainly an ABI compatibility that does not affect the possibility to login.
About ASA usually is quite picky on the server name used, it has to match its name, otherwise it refuses the connection. Specifically have a looks at `ASA database` in https://www.freetds.org/userguide/freetdsconf.html.
